### DISCUSSION TITLE
Hide Recommended Channels Row from the Following Tab

### DISCUSSION THREAD

**Discussion by LadyHaywood** [night/betterttv#6534] [Ideas]
I've seen where you can hide them from the side bar but there isn't an option to hide the row that in the following tab (unless I'm over looking a setting)

**Comment by night** [comment]
Enable the "Show Directory Live Tab" setting (Settings → Directory) — when you click Following, it'll auto-route you to /directory/following/live which skips the recommended channels row entirely. Closing as resolved.
